dist/index.html

```diff
@@ -104,6 +108,44 @@ <h2 class="steps-header">ACTIVITY</h2>
           <h2 class="friends-header">Friends</h2>
           <section class="friends-card"></section>
         </section>
+        <section class="add-form-container">
+          <section class="adding-data-section">
+          <form class="adding-data-form" action="">
+              <label class="selection-label" for="userActivitySelect">SELECT CATEGORY </label>
+              <select class="categories-value" name="categories" id="userActivitySelect">
+                  <option value=""> --Please Select An Option --</option>
+                  <option value="sleep"> Sleep Data</option>
+                  <option value="hydration"> Hydration Data</option>
+                  <option value="activity"> Activity Data</option>
+              </select>
+
+                <section  class="sleep-data-inputs hidden">
+                <label for="hoursSlept">Hours Slept <input id="hoursSlept" type="text"></label>
+                <label for="sleepQuality">Sleep Quality <input id="sleepQuality" type="text" ></label>
+              </section>
+              <section  class="hydration-data-inputs hidden">
+                <label for="numOfOunces">Number Of Ounces <input id="numOfOunces" type="text"></label>
+              </section>
+              <section  class="activity-data-inputs hidden">
+                <label for="minutesActive">Minutes Active <input id="minutesActive" type="text"></label>
+                <label for="numOfSteps">Number of Steps <input id="numOfSteps" type="text"></label>
+                <label for="flightsOfStairs">Flights Of Stairs <input id="flightsOfStairs" type="text"></label>
+              </section>
+              <section class="date-input hidden">
+                <input class="calender"
```

Comment:
The calendar will always be shown correct?

Reply:
it will show with the select they select in the dropDown